### DISCUSSION TITLE
Any feedback on linting the codebase?

### DISCUSSION THREAD

**Discussion by cclauss** [SKbarbon/flixy#3] [General]
Any feedback on #1?

**Comment by SKbarbon** [comment]
I am not great with GitHub. Can you till me what is this do ?

**Comment by cclauss** [comment]
It would help the readability of the codebase if `Reformat Code`, `Check Style`, and most importantly `Analyze (pyflakes)` were run under Pythonista's `wrench` 🔧 icon.

<img width="319" alt="Screenshot 2023-04-10 at 07 40 05" src="https://user-images.githubusercontent.com/3709715/230834453-4a7a959e-c0ff-4c41-907c-3bdc7f99d4a5.png">

**Comment by cclauss** [comment]
Adding this pull request will add suggestions to the code on GitHub.
https://github.com/internetarchive/openlibrary/pull/7766/files#annotation_10378577889
